config/channel/distributed/README.md

```diff
@@ -121,6 +121,13 @@ cluster.
                 -----END CERTIFICATE-----
   ```
 
+    > Remember to change **Net.TLS.Enable** and/or **Net.SASL.Enable** to "true" if you are
+    > configuring authentication in the [kafka-secret.yaml](300-kafka-secret.yaml) file.
+    > Also be aware that the e2e tests require those values to be false, so if you change the
+    > defaults in the [eventing-kafka-configmap.yaml](200-eventing-kafka-configmap.yaml) file
+    > itself and commit it to the repository, the tests will fail if you do not also make
+    > changes to the [e2e-tests.sh](../../../test/e2e-tests.sh) file to compensate.
+
```

Comment:
I would make the language stronger by just stating "don't commit these as true or the e2e tests will fail" and remove all the "what to do if you do so" language.

Reply:
Good idea; removed extra text.